src/components/Autocomplete/README.md

```diff
@@ -301,6 +301,123 @@ class AutocompleteExample extends React.Component {
 }
 ```
 
+### Autocomplete with lazy loading
+
+```jsx
+class AutoCompleteLazyLoadExample extends React.Component {
+  paginationInterval = 25;
+
+  options = Array.from(Array(1000)).map((_, index) => ({
```

Comment:
```suggestion
  options = Array(1000).map((_, index) => ({
```

`Array.from` shouldn't be necessary here as `Array(1000)` already returns an array data structure

Reply:
Array(number) returns an array with its length set to the number passed, but it remains an empty array. Passing it to Array.from() is a hack to turn that empty array into an array with that length and undefined at each index so you have something to map over and set whatever you actually want at each index.

Reply:
Weird. I was not aware of that hack/behavior. TIL. Thanks for the explanation!